lib/spack/spack/database.py

```diff
@@ -944,9 +1057,13 @@ def get_by_hash_local(self, dag_hash, default=None, installed=any):
             dag_hash (str): hash (or hash prefix) to look up
             default (object, optional): default value to return if dag_hash is
                 not in the DB (default: None)
-            installed (bool or any, optional): if ``True``, includes only
-                installed specs in the search; if ``False`` only missing specs,
-                and if ``any``, either installed or missing (default: any)
+            installed (bool or any, or InstallStatus or iterable of
```

Comment:
This comment is repeated 3 times: could two of them redirect to one of the others?

Reply:
I prefer having the docstrings for each function fully local. Also, the three instances should not be the same, at least two of them have different default values. I'll make sure my documentation reflects that.